Problem1.java

```diff
@@ -0,0 +1,38 @@
+//Project Euler problem 1
+//Print the sum of all multiples of 3 and 5 beneath 1000
+
+import java.util.ArrayList;
+
+public class Problem1 {
+
+  public static void main(String[] args) {
+    int i = 1;
+    int mult = 3;
+    ArrayList<Integer> thoseMults = new ArrayList<Integer> (0);
```

Comment:
Well, I guess instead of having the list, you could have just the total variable, then after all multiples of 3 and all multiples of 5 are added to the total (including duplicates), you could have a loop that will have any number that's both modulo 3 and 5 remove its own value from the total.